Content.Server/Fluids/EntitySystems/AbsorbentSystem.cs

```diff
@@ -117,30 +117,43 @@ private bool TryTransferAbsorber(EntityUid user, EntityUid used, EntityUid targe
         if (!_solutionSystem.TryGetRefillableSolution(target, out var refillableSolution, refillable: refillable))
             return false;
 
-        if (refillableSolution.Volume <= 0)
```

Comment:
Why was this inequality changed?

Reply:
it adds the ability to interact a dirty mop with an empty bucket, it just won't refill it with water, but won't block you from doing so anymore

Reply:
that == in the inequality alone would block it

Reply:
that means you can, if you want, have a mop initially empty just for the dirty solution and one filled with water just for wetting the mop

Reply:
feel free to check how it behaves yourself by connecting to estacaopirata.com

Reply:
I'm not sure if removing the whole if clause with it's contents would be best, I'm not sure if negative volumes are likely to occur, I just did the minimum change required

Reply:
The only reason it checks for < 0 was in case of errors as the volume should never be less than 0 so this if check is pointless now.